README.md

```diff
@@ -189,8 +189,8 @@ It is a standard Fastify plugin and you will not need to add the `listen` method
 }
 
 "scripts": {
-- "test": "tap test/*.test.js test/*/*.test.js test/*/*/*.test.js",
-+ "test": "standard && tap test/*.test.js test/*/*.test.js test/*/*/*.test.js",
++ "pretest": "standard",
+  "test": "tap test/*.test.js test/*/*.test.js test/*/*/*.test.js",
   "start": "fastify start -l info app.js",
   "dev": "fastify start -l info -P app.js",
```

Comment:
I think the standard `dev` script is to use --watch

Reply:
I am not sure if everyone will prefer that. @mcollina wdyt?

Reply:
I'm very -1 on `dev` using watch.